src/python/pants/backend/experimental/jvm/BUILD

```diff
@@ -0,0 +1,4 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
Makes sense. There are a lot of common rules that both would have to register, but we can put those in `pants.jvm`

Reply:
If the coursier is applicable to both languages though, shouldn't the code be in a "jvm" folder? (We can always make pants.backend.java also activate pants.backend.jvm automatically. The user should not need to know about the dependency. But for this PR, this code seems like common to all JVM languages, right?)

Reply:
The user should never activate `pants.backend.jvm` as it should not have its own regsiter.py file - it is purely implementation details, not end-user functionality they'd care about. It wouldn't "stand on its own".

The user should only need to register `pants.backend.{java,scala,kotlin}`, and those backends will do the proper registration of that shared util code in their `register.py` files. For example, all 3 of them might activate the `coursier` subsystem, and you only need to activate 1+ of the backends to get access to that subsystem.

If we do want to factor out this shared code, I think I'd recommend making `src/python/pants/jvm` for the shared code, and then have `src/python/pants/backend/java/register.py` activate it.

Reply:
I agree

Reply:
Done.  I moved everything in this PR to `src/python/pants/jvm`, since it's entirely Coursier rules and utilities.  Future PRs that add e.g. Java support will add a corresponding `src/python/pants/java/` and a thin experimental backend folder with a register.py